<ui version="4.0" >
 <class>CrewWidget</class>
 <widget class="QWidget" name="CrewWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Crews</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QTableView" name="crewTable" />
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="pushButton_new" >
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../qAirLog.qrc" >:/images/images/adept_install.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_delete" >
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../qAirLog.qrc" >:/images/images/adept_purge.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_revert" >
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../qAirLog.qrc" >:/images/images/button_cancel.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_submit" >
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../qAirLog.qrc" >:/images/images/button_ok.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
